backend/backends/null/null.go

```diff
@@ -0,0 +1,39 @@
+package null
+
+import (
+	"github.com/atlassian/gostatsd/backend"
+	"github.com/atlassian/gostatsd/types"
+
+	"github.com/spf13/viper"
+)
+
+const backendName = "null"
+
+func init() {
```

Comment:
Don't need to change but FYI, idiomatic go code starts from bottom and moves up. Comes from C/C++ I think.